skiko/src/commonMain/kotlin/org/jetbrains/skia/paragraph/FontRasterSettings.kt

```diff
@@ -0,0 +1,6 @@
+package org.jetbrains.skia.paragraph
+
+import org.jetbrains.skia.FontEdging
+import org.jetbrains.skia.FontHinting
+
+data class FontRasterSettings(val edging: FontEdging, val hinting: FontHinting, val subpixel: Boolean)
```

Comment:
Could you add a smoke test to `ParagraphStyleTests`? Just set and get the values.